packages/core/admin/admin/src/components/LeftMenu/index.js

```diff
@@ -80,12 +80,20 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
     }
   };
 
+  const menuTitle = formatMessage({
```

Comment:
When using this translation it will show `Content Manager` so you should use the one you have added in the en.json:

`app.components.LeftMenu.navbrand.title`

Reply:
Thanks for spotting that error!